Integrations/integration-Cybereason.yml

```diff
@@ -0,0 +1,536 @@
6F7Rb5CQAgrbyBAwYAZOkOnMB9lMYwGbrTXuYd7AA8Zeq4XPmDfKmPzaGIFI5JAa8+M6Fc50yr1Mmzxqko4GWjyAOvbejQYQ+sYJb7KKN3e37q7o/o8Tdb9cDqNv22pctOb3f2rTX+5P40klt/7hmmHs7n5Z2vbWQGIz8BAKSVNXDAAIAs3YETuI/iOA60l5mRai9D6D7InaY+mGt/1PZ2M/0Rc/YszSgN6KK6ci2YWqljpo2Vj5YzOe2Ke5frhnVtqjfLnVnuo2Ogd/tTp8/SCTOZ3Z7LIrGEnl3dqofXtOvmpm1q6A7Zw2iutY0ZrM/U8aZey+flPjhsn/WLhVqz6Im8C9st8hMAQFo5AwcMAMjSHTiB+yiP5ZCe7m6f6npbCN2Trjf1b7n4h9lZ73YGZls4mpyZ6XHpiLJCnTWhTCdOrtAxUyuZ+Z6DXt/YoUNtL3efR1VuZrnvKWa3576u/oieX9uupzZ26u8t3XrVtoyxrboKfKoy29HAvjQHrVMybG/M5+Wf721kBiM/AQCklTFwwACALN2BE7jvhfEc2tPdznTfQuiedIWpn+XyH2gDo15T3YPC91klBXrf2GIdXVumoyaXa2p1KWtCjkj2cmeW+2gYmN3+5OmzdCKz23PGW81denHjNr3Q1KW/tfZpbU/QeUeQ/B6V+r2yV8CI5PYQPG3qlNz/M98dYfuOyE8AAGnlCxwwACBLd+AE7ntpTAfPdLc93X2E7m873dRD+fCHbg/fIzHJBvD26VLAq7PLC3RkVbHmjy/V3NpSTa0uYa3IUks2dmq+M8vdzSz3PcDs9tzxZlOXljb3aElLt15o69NDXcHki4/20Oj35vpM9qF+Y+rT+b5OELa/E/kJACCtbIEDBgBk6Q6cwH0vjuvgnu73my899HR/2wGmnjA1OZ/+aBsyNdtwwV5sNRpPBlABn04vCWhuRYFmjinSzLFFmmZqYlUxa0mW+OJfl+kX69tVVxrIxb7T+8TA7PYnTpulk2Yxuz1bbGzt1dq2Pr3Z2qdV7X1a0tmvR3tCb7/A6HU7M9lr3O58nN5t39F1Q76vI/Rs3znyEwBAWrkCBwwAyNIdOIH7Xh9bjzlGbu/pTug+mJ0O/DdTZ+TzILTZwCFiKhpLBlT2QqsFPp1e6Ne0Er/qSws0uTyg2tKAasoCGmc+jikJsPZkkBfXtunoB1c6wSKz3IdvYHb7RRPKdeuH5zEgGabdLJutXSE1d4fU0BXUpm0hbTCfv2W+/6i9YHTIVCyRfAHR6+HdHmZElHwn15P5vu4MDtuZ2b4j8hMAQFqZAgcMAMjSHTiB+z4Y3yEXUqWn+1DXmPomw5DUa55TBW14FYubiklxJYMsj9u5OOfBfo/qA16N85sq8KrSVJn5usx8XRLwqND8TJGpgNfl3KzAfF7gczsfA163vOZ+AuZz59/Mbdwj3AWEojHz8BIKm49h+zGS/NgfjjqvG7h2CBakQvO4feZ3+81jKDSPp6jAb77Onf3PZ+9Zpl9vYJb7iNf5/ogeWTBTp86uzpm/K2K2h75gWP2RuNlO4s62Yj8fvM0NbCuFZlv0m+3Bb7dTs534zHbiMV8HvCM7TkRT26a9/5DZJu1jCZnHYLfboHkMQbO92l1LKGK2WfO9PvN1TyimrlBU20x1mGoNRrXV3Had+bjceVdOLLlfsndq83SPx5nBbrfrMlbjAYuV7Nfelu8DQRuZd0d+AgBIK0/ggAEAWboDJ3DfZ2P89kx3errvhA0o7Gx3pm7vgtNyw5YN4+O24smvB3ILZ0Jpanu2id5Aqmdnmpr/V5j1sMx8XmK+XeFxO/9cYsN38zFs7qbIfF5kvhnXzp/T2fi+LxZXv/ndfnObbhvomd/fYz7ax9Znvt8WTz22nTH3XWFuV2w+2tmvY71uVZgq83k0xm/KefHAp7FFPlWXBFTtzOj3q6q0ICuWz9NvbtWJD69y3p1QxX41bYWmNveFdEF1me64aH7WPO7WrqCatgW1tSesLaZa+yNqtxWKqjMcU1ckpm3RuFpN2Xex9JrtonPw9rqT7aPKlN0Gq00VmvKb9ajU43K2T/t1sdle4rs453Kb7bPXbJ9B83t85mfti3a95vfazbHNbrfmY5f5evtjGAg846n9inO3Q/Yndj129h+u5DtvzNes2+/qf019nmEYOrP9cq1d9E/C9iHITwAAaWUJHDAAIEt34Jw879NxtqF79bQZaqSn+86UK3kx1WMYiuELmwqm1rGgcza//aw+9TH1n8TQ7w35ud2vzKmPg752pb6xu93J0MdgPw68cJDQ29N97Wx+U1P9Xh1Y4FV9kV+TSv2aXFagKZVFOqCyULXmY6b55F1LdePmDtWVMMt9OOttd29ID5w8XWfNm5Bxj6+hvVfr2vq1rr1fG207lZ6wNvSF9WYwqrXhVCuVgVnfroFKBdWuPdw+drYdp7t97m4bTf1cQerrmPnavrTlZ5UcCbtwPmTqLoZiSNh+w0KtfY6Z7TtdachPAADpPLXjgAEAWboDJ3Dfx2OdbC9TPW2mGmkvsyu0mMljNoCN2BcNBs/mH9paJ+DVWcV+zSoJaGZlkQ4cV6wDx5dofEXhfn3sT6zaqpMfXmn7gzATOA3J2e1hnT22WPdfcuh+fzxNnf1a0dStN+1FQNv7tKI7pIfM41Mo+s5WKqlZ3wVmOftMEVTnrVdNnWWqmaEgbB8O8hMAQFoZAgcMAMjSHTih0H4Y71RP9+kz1fDNe1OhOzPdhzjK1P2mqhkKDOa01nH629sLzca3X2i2oMCns0sLNHdMoeaOL9GhdeWaMq54nz++i29frNuaupjlnu7y7AnprpOm6vxD6/b5717d3KUlTT1asqVHb7T36W7zWBQccgFQ590WtFLBTn1XvDi8HW1khof8BACQVn7AAQMAsnQHToiw38Z8h/Yybq/qepnpvhN/NHUpw4B3s/1Cs/aijjaEt5tYwKuzywp0RFWxDp9QqiPrx2h8+d7vB//gG8
```

Comment:
It is actually already displayed correctly.

![image](https://user-images.githubusercontent.com/26041559/28107360-2668a31a-66f0-11e7-9d50-e5d292e250ac.png)